app/js/components/table-filters/table-filters.jade

```diff
@@ -1,24 +1,10 @@
 .table-filters
-  
-  //- .flexgrid(ng-show='hasTags()', ng-model='tags')
-  //-   strong(translate) Active filters:&nbsp;
-  //-   // remove filter on click
-  //-   button.btn.button-active-filter(ng-repeat="(key, value) in tags", ng-click='removeFilter(key)')
-  //-     span(translate) {{value | translate}}
-  //-     i.fa.fa-times-circle-o(aria-hidden="true")
-
   .flex(ng-transclude)
-
-  .active-filters(ng-show='hasTags()', ng-model='tags')
+  .active-filters(ng-if='hasTags()', ng-model='tags')
     .flexgrid
       strong(translate) Active filters:&nbsp;
       // remove filter on click
       button.btn.button-active-filter(ng-repeat="(key, value) in tags", ng-click='removeFilter(key)')
-        span(translate) {{value | translate}}
+        span(translate) {{:: value | translate}}
```

Comment:
This breaks table filters. To reproduce:
1. go to inventory
1. select system health: healthy
1. select system health: affected
The system filter tag still says "healthy"